client/components/status/status.js

```diff
@@ -108,6 +183,23 @@ export default class Status extends React.Component {
             </div>
           </div>
         </div>
+        {captivePortalForm && (window.top === window.self) &&
+          <>
```

Comment:
what does this mean? I should refresh my react fu, maybe you can help me?

Reply:
It is possible that captive portal will redirect to the status page, in that case status page will go into a infinite loop where each IFrame will render a status page + a new Iframe, to break that I've added this check to execute this block only in root window.

Reply:
can we add a comment?